exercises/affine-cipher/description.md

```diff
@@ -0,0 +1,54 @@
+Create an implementation of the affine cipher,
+an ancient encryption system created in the Middle East.
+ 
+The affine cipher is a type of monoalphabetic substitution cipher.
+Each character is mapped to its numeric equivalent, encrypted with
+a mathematical function and then converted to the letter relating to
+its new numeric value. Although all monoalphabetic ciphers are weak,
+the affine cypher is much stronger than the atbash cipher,
+because it has many more keys.
+ 
+the encryption function is:
+ 
+  `E(x) = (ax + b) mod m`
+  -  where `x` is the letter's index from 0 - length of alphabet - 1
+  -  `m` is the length of the alphabet. For the roman alphabet `m == 26`.
+  -  and `a` and `b` make the key
+ 
+the decryption function is:
+ 
+  `D(y) = a^-1(y - b) mod m`
+  -  where `y` is the numeric value of an encrypted letter, ie. y = E(x)
+  -  it is important to note that a^-1 is the modal multiplicative inverse
```

Comment:
please rename `modal multiplicative inverse` to `modular multiplicative inverse`.  This makes more sense to me, and was the source of my confusion.  This change should apply to all instances.

Reply:
Yes, I realized I misspelled it `modal` instead of `modular`.  I will be home soon and can commit more fixes.